api/controllers/booksController.js

```diff
@@ -0,0 +1,188 @@
+const Sequelize = require("sequelize");
+const db = require("../../models");
+const flash = require("express-flash");
+const BookModel = db.Book;
+const pdf = require("html-pdf");
+const mailer = require("../../email/mailer");
+const getTemplate = require("../../helpers/email-template").getTemplate;
+
+let Book = {};
+
+Book.add = async (req, res, next) => {
+  try {
+    res.render("books/add", { loggedin: req.isAuthenticated() ? true : false });
+    res.status(200);
+  } catch (err) {
+    next(err);
+  }
+};
+
+Book.create = async (req, res, next) => {
+  try {
+    const user = await req.user;
```

Comment:
is await necessary in these cases to obtain the parameters? 🤔

Reply:
I think so because req.user is a promise, and if i leave it like
`const user = req.user;`
user is a promise type